src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/query/ObTableQuery.java

```diff
@@ -61,7 +61,12 @@ public class ObTableQuery extends AbstractPayload {
     private static final byte[] HTABLE_FILTER_DUMMY_BYTES = new byte[] { 0x01, 0x00 };
     private boolean             isHbaseQuery              = false;
     private List<String>        scanRangeColumns          = new LinkedList<String>();
+    
+    private List<AggregationSingle>    aggregations = new LinkedList<>();
```

Comment:
Shall we keep those in the same style?
```suggestion
    private List<AggregationSingle>    aggregations       = new LinkedList<>();
```

Reply:
fixed